e2-studio-star-rx72n-firmware/src/boot/r_bsp/r_rx_intrinsic_functions.c

```
@@ -0,0 +1,1237 @@
+  register signed long* ldata2 = (signed long*)data2;
+  /* this is much more then an "intrinsic", no inline asm because of loop */
+  /* will implement this.. interesting function as described in ccrx manual */
+  __builtin_rx_mullo(0, 0);
+  while (count > 1) {
+    __builtin_rx_maclo(*ldata1, *ldata2);
+    __builtin_rx_machi(*ldata1, *ldata2);
+    ldata1++;
+    ldata2++;
+    count -= 2;
+  }
+  if (count != 0)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)